.erda/migrations/qa/20210906-auto-test-plan-execute.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE `dice_autotest_plan` ADD `execute_time` TIMESTAMP DEFAULT NULL COMMENT 'auto test plan latest execute time';
```

Comment:
execute_time_sec

Reply:
plan execute time.
type is time.time
need not _sec